applications/ISO20022.md

```diff
@@ -0,0 +1,162 @@
+
+Please also provide the GitHub accounts of all team members. 
+- [GitHub1](https://github.com/OpenSmartContract) 
+- [Github2](https://github.com/FPLeader) 
+
+### Team LinkedIn Profiles (if available)
+- [Profile](https://www.linkedin.com/in/dengdaniel)
+
+## Development Status 
+
+- [Blockchain Food Order dApp](https://github.com/InkSmartContract/BlockchainFoodOrder/blob/main/Learn%20ink!%20by%20Example%20%E2%80%94%20Order%20Food%20on%20the%20Blockchain.pdf)
+- [Smart Contract Deployment and Interaction Guide using Contracts-UI and Polkadot.js](https://github.com/InkSmartContract/BlockchainFoodOrder/blob/main/BlockchainFoodOrder%20Smart%20Contract%20Deployment%20and%20Interactions%20Guide.pdf)
+
+## Development Roadmap 
+
+### Overview
+
+- **Estimated duration:** ~1-2 months
+- **FTE:**  1,2
+- **Costs:** 25K USDd
+
+### Milestone 1
+
+- **Estimated duration:** ~1-2 months
+- **FTE:**  1,2
+- **Costs:** 25K USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | Design documents with details on what it takes for Polkadot/Substrate to support ISO20022 messages, gaps and solution recommendations |
+| 1. | Design the use case | Degine an ISO20022 cross border payment use case with persona, end-to-end flow | 
+| 2. | Specify ISO20022 messages | Specify ISO20022 messages for each and every step of the cross border payment flow, including reference data fields besides core XML data | 
+| 3. | ISO20022 XML <> Polkadot XCM data mapping | Identify data types gaps, XML<>XCM trnasformation and validations for the use case| 
+| 4. | ISO20022 <> Polkadot messaging | Identify XCMP gaps in messaging, relaying, routing feasibilities for the use case | 
+| 5. | Define offchain data storage | Define offchain data structure, storage and indexes for ISO20022 messages and payments in this use case |
+| 6. | Define offchian worker module | Define OCW logic to handle ISO20022 messages mapping, processing, messaging in this use case |
+| 7. | Define Smart contracts | Define ink! smart contracts to implement X-border payments and offchian integrations in this use case |
+| 8. | Define ISO20022 messaging and payment events and stats | Define ISO20020 messaging and payment flow events and states |
+| 9. | Define ISO20022 messaging and payment queries | Define how to query ISO20022 messaging and payments, based off events, stats, transactions and indexes stored offchain |
+
+
+## Future Plans
+
+- Publish learning and share with the community, similar to this [ Medium article](https://medium.com/@opensmartcontract/learn-ink-by-example-order-food-on-blockchain-a4024b2dee4a) and/or [Contracts-UI and Polkadot.js Smart Contract Deployment and Interaction Guide](https://github.com/InkSmartContract/BlockchainFoodOrder/blob/main/BlockchainFoodOrder%20Smart%20Contract%20Deployment%20and%20Interactions%20Guide.pdf)
+- Long term we intend to expand this ISO20022 PoC to:
```

Comment:
How many people are going to work on this project?

Reply:
That's fine. However, those consultants should be listed in the team section as well. If there changes down the road, you'll be able to amend it later, even after the proposal would be approved.

Reply:
Noted in [revised proposal ](https://github.com/w3f/Grants-Program/pull/1940/commits/a17d45e75dda9d6c465488ce454fdb8e0446fc16)